data/json/recipes/recipe_food.json

```diff
@@ -4114,7 +4114,8 @@
     "components": [
       [
         [ "sweet_fruit_like", 2, "LIST" ],
-        [ "bum_wine", 14 ],
+        [ "cheap_wine", 14, "LIST" ],
+        [ "worthy_wine", 10, "LIST" ],
```

Comment:
There are 5g of total carbohydrates in 150ml of wine. Attempting to refine wine into sugar would be extremely inefficient.